src/chrome/content/rules/EFF.xml

```diff
@@ -1,17 +1,26 @@
-<!--
-	Other EFF rulesets:
-		- Freerangekitten.com.xml
-		- IFightSurveillance.org.xml
-		- ManilaPrinciples.org.xml
-		- Panopticlick.com.xml
+<ruleset name="EFF">
 
--->
+	<target host="freerangekitten.com" />
```

Comment:
IIRC, this domain is intentionally leave unprotected, and reserved like `example.com`. And this domain is also used in the unit tests.